packages/experimental-nextjs-app-support/README.md

```diff
@@ -5,8 +5,8 @@
 # Apollo Client support for the Next.js App Router
 
 > ❗️ This package is experimental.<br/>
-> Generally it should work well, you might run into race conditions when your Client Component is still rendering in SSR, and already making overlapping queries on the browser.  
```

Comment:
This whitespace was unfortunately very important - in GitHub markdown, two spaces at a line end mark a `<br/>` tag 😬

Reply:
Wow, thats quite a choice by them. It looks like `\` also works to add line breaks, would you be okay with a PR that uses those as the more explicit option to get the docs back to the way they were?

Reply:
@jer-k yes! Apologies on my part for not realizing the spaces were intentional. I've got a command to remove trailing whitespace when I save in my editor, so I think the more explicit `\` would be welcome :).

Feel free to open a new PR if you'd like, otherwise I can tackle this later today.